packages/react/modules/components/configure-adapter/configure-adapter.tsx

```diff
@@ -64,7 +103,7 @@ const ConfigureAdapter = (props: TProps) => {
        */
       setAppliedAdapterArgs(nextAdapterArgs);
```

Comment:
<img width="706" alt="CleanShot 2020-02-25 at 09 44 32@2x" src="https://user-images.githubusercontent.com/1877073/75230290-ac516780-57b3-11ea-8614-938db40f54b7.png">
<img width="629" alt="CleanShot 2020-02-25 at 09 44 24@2x" src="https://user-images.githubusercontent.com/1877073/75230300-aeb3c180-57b3-11ea-9906-a2bbf074a245.png">

These now blow to be not callable. I don't follow why. Am I missing something? I would expect TS to infer the return type correctly. /cc @emmenko

Reply:
Can't say much like this, I would need to check out the branch